From fc2838076c1de3bff38020187b6d1a9b18a7237c Mon Sep 17 00:00:00 2001
From: Marek Kasik <mkasik@redhat.com>
Date: Tue, 30 Jul 2013 11:07:11 +0200
Subject: [PATCH 03/12] print-notifications: Use better debugging messages

Using g_debug() is more appropriate than gnome_settings_profile_msg()
for message about getting dests from CUPS server.
Be more specific in those debugging messages.
---
 plugins/print-notifications/gsd-print-notifications-manager.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/plugins/print-notifications/gsd-print-notifications-manager.c b/plugins/print-notifications/gsd-print-notifications-manager.c
index ba38a49..954a919 100644
--- a/plugins/print-notifications/gsd-print-notifications-manager.c
+++ b/plugins/print-notifications/gsd-print-notifications-manager.c
@@ -999,7 +999,7 @@ cups_connection_test_cb (GObject      *source_object,
                 g_object_unref (connection);
 
                 manager->priv->num_dests = cupsGetDests (&manager->priv->dests);
-                gnome_settings_profile_msg ("got dests");
+                g_debug ("Got dests from remote CUPS server.");
 
                 renew_subscription (user_data);
                 g_timeout_add_seconds (RENEW_INTERVAL, renew_subscription_with_connection_test, manager);
@@ -1036,7 +1036,7 @@ cups_connection_test (gpointer user_data)
                         g_object_unref (client);
                 } else {
                         manager->priv->num_dests = cupsGetDests (&manager->priv->dests);
-                        gnome_settings_profile_msg ("got dests");
+                        g_debug ("Got dests from local CUPS server.");
 
                         renew_subscription (user_data);
                         g_timeout_add_seconds (RENEW_INTERVAL, renew_subscription_with_connection_test, manager);
-- 
1.8.3.1

